llama_stack/providers/tests/conftest.py

```diff
@@ -24,6 +28,63 @@ class ProviderFixture(BaseModel):
     provider_data: Optional[Dict[str, Any]] = None
 
 
+class Fixtures(BaseModel):
+    # provider fixtures can be either a mark or a dictionary of api -> providers
+    provider_fixtures: List[Dict[str, str]] = Field(default_factory=list)
+    inference_models: List[str] = Field(default_factory=list)
+    safety_shield: Optional[str] = Field(default_factory=None)
+    embedding_model: Optional[str] = Field(default_factory=None)
+
+
+class APITestConfig(BaseModel):
+    fixtures: Fixtures
+
+    # test name format should be <relative_path.py>::<test_name>
+    tests: List[str] = Field(default_factory=list)
+
+
+class TestConfig(BaseModel):
+    inference: APITestConfig
```

Comment:
why is inference special and non-optional? it should also be optional just like others?